packages/wonder-blocks-theming/src/tokens.ts

```diff
@@ -25,6 +25,11 @@ const tokens = {
         // Khanmigo
         white32: fade(Color.white, 0.32),
         eggplant: "#5f1e5c",
+        // Pill
+        pillBlue: fade(Color.blue, 0.24),
+        pillGreen: fade(Color.green, 0.24),
+        pillGold: fade(Color.gold, 0.24),
+        pillRed: fade(Color.red, 0.24),
```

Comment:
FYI: I did check and confirm that all of these pass AA (and AAA) contrast guidelines when they're used as the background color with black text.

Reply:
Also FYI: These colors are based on the colors in `coverage-badge.tsx` in webapp. You can see this being used on Khan Academy here:

https://www.khanacademy.org/standards/CCSS.Math/3.OA

I originally tried using the banner background colors, but I felt like they were just way too light and you couldn't even see the pill.

Reply:
Could you please change the token names here? These new colors represent semantic names and in here we are only using core/primitive tokens.

As for the color values, let me check with Design if these are the colors that we want to officially introduce to WB.

Reply:
We discussed this with the design team, and the plan is to use the 16% option.... we could change to 24% in the future, but the plan is to still with 16% for now, for consistency across the system.